src/Service/WpCronService.php

```diff
@@ -25,9 +28,47 @@ public function dispatch($callback, ...$args): void
      * @param                                  ...$args
      *
      * @return void
+     * @throws \Exception
      */
     public function schedule($callback, int $timestamp, ...$args): void
     {
-        wp_schedule_single_event($timestamp, $callback, $args);
+        $hook = $callback;
+
+        if (is_callable($callback)) {
+            $hook    = md5(uniqid('', true));
+            $actions = get_option(Pdk::get('myParcelActions'), []);
+
+            update_option(Pdk::get('myParcelActions'), $this->getActions($callback, $actions, $hook));
+        }
+
+        wp_schedule_single_event($timestamp, $hook, $args);
+    }
+
+    /**
+     * @param $callback
+     * @param $actions
+     * @param $hook
+     *
+     * @return array|string
+     */
+    private function getActions($callback, $actions, $hook)
+    {
+        if (is_string($callback)) {
+            $actions[sprintf('%s%s', Pdk::get('myParcelHook'), $hook)] = $callback;
```

Comment:
variabelen en dergelijke mag je gewoon concatenaten, de "niet concatenaten" regel geldt alleen wanneer je ook echt strings in de mix hebt